js/components/dataset/quality.vue

```diff
@@ -12,7 +12,8 @@
 
     <h3>{{ _('Tags') }}</h3>
     <p>{{ _('Tags helps your reusers to find your resources, that is very important to make your dataset popular.') }}</p>
-    <p>{{ _('You currently set {tags_count} tags for that dataset.', {tags_count: quality.tags_count}) }}</p>
+    <p v-if="quality.tags_count">{{ _('You currently set {tags_count} tags for that dataset.', {tags_count: quality.tags_count}) }}</p>
+    <p v-if="!quality.tags_count" class="text-warning fa fa-exclamation-circle"> <strong> {{ _('You currently set no tags for that dataset! Why not add some more?') }}</strong></p>
```

Comment:
In order to have a space between the icon and the text, you should write it like that:
```html
<p v-if="!quality.tags_count" class="text-warning">
<span class="fa fa-exclamation-circle"></span>
<strong>{{ _('You currently set no tags for that dataset! Why not add some more?') }}</strong>
</p>
```